skfuzzy/control/fuzzyvariable.py

```diff
@@ -13,8 +15,32 @@
 except ImportError:
     from .ordereddict import OrderedDict
 
+class TermPrimitive(object):
+    """Marker class used for type checking when a term
+    or term aggregate is expected"""
+    pass
 
-class FuzzyVariableTerm(object):
+    def membership_value(self):
```

Comment:
While not strictly needed in python, I usually add these stubs to abstract members of abstract classes to make it more clear to anyone who does a future extension of the class what they should be implementing in the concrete class.  Another way would be to use the `abstractpropery` decorator in the `abc` module, but I think this also gets the point across without using the `abc` metaclass.